packages/lib/src/components/BankTransfer/BankTransfer.tsx

```diff
@@ -42,6 +42,10 @@ export class BankTransferElement extends UIElement<BankTransferProps> {
     };
 
     render() {
+        if(this.props.showEmailAddress === undefined || this.props.showEmailAddress === null){
+            this.props.showEmailAddress = BankTransferElement.defaultProps.showEmailAddress;
+        }
+
```

Comment:
We can safely assume `this.props` will already contain any `defaultProps` defined in the class (check the constructor on `BaseElement`)

Reply:
Well seen! Mecí, I have forgot to check that class